lib/perf_hooks.js

```diff
@@ -395,12 +395,14 @@ class Performance {
 
   measure(name, startMark, endMark) {
     name = `${name}`;
-    endMark = `${endMark}`;
-    startMark = startMark !== undefined ? `${startMark}` : '';
     const marks = this[kIndex][kMarks];
-    if (!marks.has(endMark) && !(endMark in nodeTiming)) {
-      throw new ERR_INVALID_PERFORMANCE_MARK(endMark);
+    if (arguments.length === 3) {
```

Comment:
```suggestion
    if (endMark !== undefined) {
```

to match with the check for `startMark` below?

Reply:
I'm worried someone who calls `measure(xxx, xxx, undefined)`, maybe a breaking-change

Reply:
btw, test added

Reply:
@Himself65 I think that would be surprising, given that we already have a similar check for `startMark`

Reply:
after thinking twice, I think you are right

Reply:
unfortunately, there will throw an error if we do this (maybe a breaking-change). so I still use `arguments.length === 3`

https://github.com/nodejs/node/blob/e038d6a1cdb2ffbf666d360553c31786acaf70b2/test/sequential/test-perf-hooks.js#L35-L43